src/containers/Identity/PersonalInfo/ClaimManager/MoveIntoCategory/styles.js

```diff
@@ -0,0 +1,13 @@
+import { StyleSheet } from 'react-native';
+
+
+export default StyleSheet.create({
+  root: {
+    flex: 1,
+    padding:16,
+  },
+  text:{
+    color:'#b5b5b5',
```

Comment:
Think about extracting the colors in a top theme styles file, so we can adjust it in one place. Of course if this is a color that is dependent of the theme.